nbs/03_Data_Exploration.ipynb

```diff
@@ -0,0 +1,903 @@
+{
```

Comment:
<p>Avoid <code>%cd</code> in code directly. Instead, use the Path(data_dir)/"filename" kind of convention?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>